cfme/common/provider_views.py

```diff
@@ -248,5 +286,146 @@ def is_displayed(self):
 
     toolbar = View.nested(ProviderToolBar)
     sidebar = View.nested(ProviderSideBar)
-    entities = View.nested(ProviderEntities)
+    items = View.nested(ProviderItems)
     paginator = View.nested(PaginationPane)
+
+
+class BeforeFillMixin(object):
+    def before_fill(self):
+        if self.exists and not self.is_active():
+            self.select()
+
+
+class DefaultEndpointForm(View):
+    hostname = Input('default_hostname')
+    username = Input('default_userid')
+    password = Input('default_password')
+    confirm_password = Input('default_verify')
+
+    validate = Button('Validate')
+
+
+class SCVMMEndpointForm(DefaultEndpointForm):
```

Comment:
sorry, my error.  Endpoints and EndpointForms have been recently moved to appropriate provider files in #4535. I just forgot to rebase this PR against that one.